etc/scripts/docenizers/docenizer-python.py

```diff
@@ -97,7 +115,7 @@ def main():
     args = parser.parse_args()
     print(f"Called with: {args}")
     # If we don't have the html folder already...
-    if not os.path.isdir(os.path.join(args.inputfolder, 'html')):
+    if not os.path.exists(os.path.join(args.inputfolder, ARCHIVE_NAME)):
```

Comment:
why the change from isdir? in this instance I don't think it matters but just wondering.

Reply:
(Sorry for the slow response. I've been away for a week.)

The code didn't create any directory. It only downloads a file. I could have used `os.path.isfile` I think.